config/default/app_settings.json

```diff
@@ -343,9 +343,9 @@
);n||(n=r(t.reported_date));const i=d(e,l,n.toDate(),n.clone().add(294,'days').toDate()),s=[];i.forEach(e=>{D(e)?'yes'===u(e,'pregnancy_summary.visit_option')&&s.push(e):s.push(e)});const a=c(s,l);return a?p(a):[]},getNextANCVisitDate:function(e,t){let n=u(t,'t_pregnancy_follow_up_date'),i=t.reported_date;return T(e,t).forEach((function(e){e.reported_date>i&&u(e,'t_pregnancy_follow_up_date')&&(i=e.reported_date,n=u(e,'t_pregnancy_follow_up_date'))})),r(n)},isReadyForNewPregnancy:function(e,t){if('person'!==e.type)return!1;const n=c(t,s),a=c(t,o);if(!n&&!a)return!0;if(n){if(!a||a.reported_date<n.reported_date){let e=f(t,n);if(e||(e=r(n.reported_date)),e<i.clone().subtract(294,'day'))return!0;if(b(t,n,'abortion')||b(t,n,'miscarriage'))return!0}else if(n&&b(t,n,'abortion')||b(t,n,'miscarriage'))return!0}else if(a&&m(a)<i.clone().subtract(42,'day'))return!0;return!1},isReadyForDelivery:function(e,t){if('person'!==e.type)return!1;const n=c(t,s),a=c(t,o);if(!n&&!a)return!0;if(a&&(!n||a.reported_date>n.reported_date))return m(a)<i.clone().subtract(7,'months');if(n&&k(n)){const e=f(t,n);if(e){const r=_(t,n);return i.isBetween(e.clone().add(6,'months'),r.clone().add(6,'weeks'))}return r(n.reported_date).clone().startOf('day').add(322,'days').isSameOrBefore(i)}return!1},getMostRecentLMPDateForPregnancy:f,getMostRecentEDDForPregnancy:_,getDeliveryDate:m,getFormArraySubmittedInWindow:d,getRecentANCVisitWithEvent:b,getField:u}},function(e,t){function n(e){return e?Array.isArray(e)?e:[e]:[]}function r(e,t){switch(typeof e){case'undefined':return!0;case'function':return e(t);default:return e}}function i(e,t,n){if(r(e.appliesIf,n)){var i='function'==typeof e.fields?e.fields(n):e.fields.filter((function(e){return r(e.appliesIf,n)})).map((function(e){var t={};return s(e,t,'label'),s(e,t,'value'),s(e,t,'translate'),s(e,t,'filter'),s(e,t,'width'),s(e,t,'icon'),e.context&&(t.context={},s(e.context,t.context,'count'),s(e.context,t.context,'total')),t}));return e.modifyContext&&e.modifyContext(t,n),{label:e.label,fields:i}}function s(e,t,r){switch(typeof e[r]){case'undefined':return;case'function':t[r]=e[r](n);break;default:t[r]=e[r]}}}e.exports=function(e,t,r){var s=e.fields||[],a=e.context||{},o=e.cards||[],l=t&&('contact'===t.type?t.contact_type:t.type),u={cards:[],fields:s.filter((function(e){var t=n(e.appliesToType),r=t.filter((function(e){return e&&'!'===e.charAt(0)}));if((0===t.length||t.includes(l)||r.length>0&&!r.includes('!'+l))&&(!e.appliesIf||e.appliesIf()))return delete e.appliesToType,delete e.appliesIf,!0}))};return o.forEach((function(e){var t,s,o,d,c=n(e.appliesToType);if(c.includes('report')&&c.length>1)throw new Error('You cannot set appliesToType to an array which includes the type \\'report\\' and another type.');if(c.includes('report'))for(t=0;t<r.length;++t)s=r[t],!(d=s)||d.errors&&d.errors.length||(o=i(e,a,s))&&u.cards.push(o);else{if(!c.includes(l)&&c.length>0)return;(o=i(e,a))&&u.cards.push(o)}})),u.context=a,u}}])})); return ContactSummary;",
```

Comment:
This block is hard to verify in code review. My only real check is that there's a decent amount less code in the rules section now. Does that seem right to you?

Reply:
I'm having another look. 

I added the `check-app-settings-is-committed` to the testing scheme for `default` as well (it was only running for `standard`), so the test says it's ok :D 
https://travis-ci.com/github/medic/cht-core/jobs/491376417#L14844

Reply:
Ok, I checked the two sources and compared char count: 

`contact_summary`: 
- old: 69679 chars
- new: 69614 chars
- diff: -65 chars 

`rules`:
- old: 18289 chars
- new: 18321 chars
- diff: -32 chars

I think the visual differences are caused by wrapping.

Reply:
👍